README.md

```diff
@@ -133,12 +133,122 @@ If you have all of your components in the folder `./comps` then you would call `
 
 Every folder directly under `./comps` would get processed and the build tools and handed off to rollup.
 
+## Locale files
+
+Locale files are JSON files that contains locale specific translations of strings you use in your application. By default the files are all placed in the `locales` folder and are names `strings_??.json` where the `??` is the 2 letter locale such as `en`, `fr`, or `ja`.
```

Comment:
`contains` should be `contain`
`names` should be `named`

Reply:
Fixed